frontend/src/pages/CompaniesList/index.tsx

```diff
@@ -54,11 +54,11 @@ export const CompaniesList: FC = () => {
       render: (_, record) => {
         return (
           <Space size="middle">
-            <Button className="invbutton" onClick={() => handleVote(record)}>
+            <Button className="inverted_button" onClick={() => handleVote(record)}>

```

Comment:
why inverted_buttons?  If you ask someone to read the code, will that person be able to understand that this class is for these buttons?
Suggestions: action_buttons, buttons, etc